modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/OperationModelLoaderDelegate.java

```diff
@@ -55,6 +58,8 @@
 final class OperationModelLoaderDelegate extends AbstractModelLoaderDelegate {
 
   private static final String OPERATION = "Operation";
+  private static final ClassTypeLoader TYPE_LOADER = ExtensionsTypeLoaderFactory.getDefault()
```

Comment:
Weird:
a) added but never used
b) shoudln't this use the extension's classlaoder?

Reply:
no action?